tests/test_simple.py

```diff
@@ -309,3 +309,60 @@ def test_exception():
         response = client.post("/predict", json={"input": 4.0})
         assert response.status_code == 500
         assert response.json() == {"detail": "Internal server error"}
+
+
+def test_shutdown_endpoint():
+    """Test the shutdown endpoint with API key authentication."""
+    # Create server with shutdown endpoint enabled
+    server = LitServer(
+        SimpleLitAPI(),
+        accelerator="cpu",
+        devices=1,
+        workers_per_device=1,
+        enable_shutdown_api=True,
+        shutdown_path="/shutdown",
+    )
+
+    # Mock the server state to avoid actually shutting down during tests
+    from types import SimpleNamespace
```

Comment:
please move the import to top

Reply:
thanks! @Gurudeep-hn, can you please check the windows failing test please - https://github.com/Lightning-AI/LitServe/actions/runs/15436642205/job/43450072384?pr=524